apps/condo/utils/clientSchema/search.ts

```diff
@@ -0,0 +1,43 @@
+import {
```

Comment:
@pahaz I want to introduce  `clientSchema/serach` module which will include all search functions needed

Reply:
I have some thoughts on this.